third-party/arcanum-worlds/odyssey-of-the-dragonlords/items.xml

```diff
@@ -37,4 +35,707 @@
+			<p class="indent">Titansbane is sentient and wishes to be returned to its master. It was forged by Talieus, the ancient titan who pulls Lutheria’s barge. It was wielded against Sydon by Hergeron and then lost in the War of the Titans.</p>
+		</description>
+		<setters>
+			<set name="keywords">longsword</set>
+			<set name="category">Magic Weapons</set>
+			<set name="cost" currency="gp">0</set>
+			<set name="type" addition="longsword">Weapon</set>
+			<set name="attunement">true</set>
+			<set name="rarity">Legendary</set>
+			<set name="enhancement">2</set>
+			<set name="weapon">Longsword</set>
+		</setters>
+	</element>
+	<element name="Xiphos of Slaughter" type="Magic Item" source="Odyssey of the Dragonlords" id="ID_AW_OOTD_MAGIC_ITEM_XIPHOS_OF_SLAUGHTER">
+		<description>
+			<p>Centuries ago, the Dragonlord Estor Arkelander defeated a gygan smith in single combat and forced the cyclops into his service. He demanded that the cyclops forge him a weapon that would drench all the shores of Thylea in blood. The gygan complied, crafting this bronze dagger. Estor anointed the weapon by slaughtering the smith and his family. The weapon is stained deep red.</p>
+			<p class="indent">You gain a +2 bonus to attack and damage rolls made with this magic weapon.</p>
+			<p class="indent"><b>Cursed.</b> This dagger is cursed, and becoming attuned to it extends the curse to you. As long as you remain cursed, you are unwilling to part with the dagger, keeping it within reach at all times. You also have disadvantage on attack rolls with weapons other than this one.</p>
+			<p class="indent">At the beginning of your turn in combat, you must use your actions to attack the nearest creature at least once. If you have multiple possible targets, you attack one at random. At the end of your turn in combat, if you did not inflict damage on any creature this turn, you suffer 1d10 psychic damage.</p>
+		</description>
+		<setters>
+			<set name="keywords">dagger</set>
+			<set name="category">Magic Weapons</set>
+			<set name="cost" currency="gp">0</set>
+			<set name="type" addition="dagger">Weapon</set>
+			<set name="attunement">true</set>
+			<set name="rarity">Very Rare</set>
+			<set name="enhancement">2</set>
+			<set name="weapon">Dagger</set>
+		</setters>
+	</element>
+
+	<!-- Armor -->
+	<element name="Breastplate of Mytros" type="Magic Item" source="Odyssey of the Dragonlords" id="ID_AW_OOTD_MAGIC_ITEM_BREASTPLATE_OF_MYTROS">
+		<description>
+			<p>A character has a +1 bonus to AC and a +1 bonus to Charisma while wearing this armor. Giants and titans have disadvantage on their attacks against an opponent wearing the Breastplate.</p>
+		</description>
+		<setters>
+			<set name="keywords">breastplate</set>
+			<set name="category">Magic Armor</set>
+			<set name="cost" currency="gp">0</set>
+			<set name="weight" lb="0">0 lb.</set>
```

Comment:
with magic armor the **weight** setter has to be left out unless you want to override the base weight